src/generators/css-filter.ts

```diff
@@ -63,7 +76,8 @@ export function cssFilterStyleheetTemplate(filterValue: string, reverseFilterVal
     });
 
     if (!frameURL) {
-        const [r, g, b] = applyColorMatrix([255, 255, 255], createFilterMatrix(config));
+        const [x, y, z] = hasChromiumIssue501582() ? [0, 0, 0] : [255, 255, 255];
```

Comment:
Well, `[x, y, z]` looks not clear. Also you are destructuring an array into parts, and later put these parts into another array. Could you please use a variable with a name explaining what is `[0, 0, 0]` or `[255, 255, 255]`?